g/goproxy/goproxy_ubi_8.4.sh

```diff
@@ -0,0 +1,71 @@
+# ----------------------------------------------------------------------------
+#
+# Package               : elazarl/goproxy/ext
+# Version               : v0.0.0-20190421051319-9d40249d3c2f
+# Source repo           : https://github.com/elazarl/goproxy.git
+# Tested on             : UBI 8.4
+# Language              : GO
```

Comment:
looks like the TravisCheck field as per the latest template is missing?

Reply:
Added TravisCheck field.